charts/zeebe-operate-helm/templates/deployment.yaml

```diff
@@ -35,10 +42,19 @@ spec:
         - name: config
           mountPath: /usr/local/operate/config/application.yml
           subPath: application.yml
+        {{- if .Values.extraVolumeMounts}}
+        {{ .Values.extraVolumeMounts | toYaml | nindent 8 }}
+        {{- end }}
       volumes:
       - name: config
         configMap:
           name: {{ include "zeebe-operate.fullname" . }}
           defaultMode: 0744
+      {{- if .Values.extraVolumes}}
+      {{ .Values.extraVolumes | toYaml | nindent 6 }}
+      {{- end }}
+      {{- if .Values.serviceAccount.name}}
+      serviceAccountName: {{ .Values.serviceAccount.name }}
```

Comment:
What would be the use case of this?

Reply:
A service account to catch secrets from Vault for example ? and not use default SA